lib/settings.js

```diff
@@ -8,6 +8,7 @@ module.exports = settings => {
       level: 'info',
       format: 'dev'
     },
+    verboseErrors: true,
```

Comment:
Will this apply for all envs at the min?

Reply:
yeah

Reply:
we can override it by setting verbose: false at the implementation level

Reply:
I'd default to false ideally, and then override for dev environments.

The fallback options should try to be "prod safe" where possible.

Reply:
good point sir